packages/api-client/src/views/Request/RequestSection/RequestBody.vue

```diff
@@ -319,12 +319,23 @@ const updateActiveBody = (type: Content) => {
     }
   }
   // Add header if doesn't have one
-  else if (header)
-    headers.push({
-      key: 'Content-Type',
-      value: header,
-      enabled: true,
-    })
+  else if (header) {
+    const lastHeader = headers[headers.length - 1]
+    // Add header before last if empty to prevent empty row duplication
```

Comment:
very nice, but now I'm thinking should we just add this to the import? would be a little cleaner. What do you think

Reply:
we could but we would still need logic for when we select a body type let's say get -> post to populate it right?

Reply:
ya true